lib/liquid/expression.rb

```diff
@@ -29,5 +31,18 @@ def self.parse(markup)
       end
     end
 
+    def self.format(value)
+      if INVERTED_LITERALS.key?(value)
+        INVERTED_LITERALS[value].dup
+      elsif value.is_a?(VariableLookup) || value.is_a?(RangeLookup)
+        value.format
+      elsif value.is_a?(String)
+        "\"#{value}\""
```

Comment:
This should probably be formatted with single quotes if the string contains double quotes.